src/gluonts/ext/r_forecast/R/hierarchical_forecast_methods.R

```diff
@@ -1,4 +1,5 @@
 library("hts")
+library(parallel)
```

Comment:
Are quotation marks not needed? (The previous line has them, but I’m not fluent in R)

Reply:
I updated it to follow the quotation marks style.